packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -9,8 +9,92 @@
 >
 > [CFN Resources]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib
 
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
 ---
 
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+[AWS Service Catalog](https://docs.aws.amazon.com/servicecatalog/latest/dg/what-is-service-catalog.html)
+enables organizations to create and manage catalogs of products for their end users that are approved for use on AWS.
+
+## Table Of Contents
+
+- [Portfolio](#portfolio)
+  - [Granting access](#granting-access-to-a-portfolio)
```

Comment:
Why are the headings in the ToC different than the actual headings...? Is it just a miss, or is there some idea behind it...?

Reply:
Was trying to keep the main part of the title shorter.  Otherwise we would have short header and then have to repeat it in the first sentence with brief description.